pkg/framework/context/context.go

```diff
@@ -188,6 +199,14 @@ func WithProtocolServices(services ...dispatcher.Service) ProviderOption {
 	}
 }
 
+// WithGenericInboundServices injects services for handling generic inbound messages.
+func WithGenericInboundServices(services ...dispatcher.GenericInboundService) ProviderOption {
```

Comment:
I am guessing that this `GenericInbound` is not the most intuitive. I am guessing you are trying to differentiate between a full protocol service and a message service?

Reply:
`WithMessageServices`, `WithMessageHandlers`, ...?

Reply:
`MessageHandler` is wildly used, and may be confusing with https://github.com/hyperledger/aries-framework-go/blob/master/pkg/didcomm/transport/transport_interface.go#L33

Will go go `MessageService` for now,

Reply:
ok

Reply:
done